code/common/datastripe.q

```diff
@@ -1,3 +1,26 @@
+// stripe data across rdbs
+
 \d .ds
 
 segmentid: "J"$.proc.params[`segid]		// segmentid variable defined by applying key to dictionary of input values
+
+\d .
+
+.rdb.datastripeendofperiod:{[currp;nextp;data]
+    .lg.o[`reload;"reload command has been called remotely"];
+    
+    // remove periods of data from tables
+    t:tables[`.] except .rdb.ignorelist;
```

Comment:
Why are we calling this like tables[`.]

Reply:
this function is in the .ds namespace so tables[] would get all tables in .ds namespace. need to call tables[`.] to get the tables in the root namespace instead